src/wp-includes/rest-api/endpoints/class-wp-rest-post-types-controller.php

```diff
@@ -246,6 +246,14 @@ public function prepare_item_for_response( $item, $request ) {
 			$data['rest_namespace'] = $namespace;
 		}
 
+		if ( rest_is_field_included( 'template', $fields ) ) {
+			$data['template'] = ! isset( $post_type->template ) ? null : $post_type->template;
```

Comment:
```suggestion
			$data['template'] = $post_type->template ) ?? null;
```

Can we use here `??`

Reply:
Hi end-up using the same logic used at https://github.com/WordPress/wordpress-develop/blob/5c7f94a6aa979dc70ff935e9006957fe0f262932/src/wp-admin/edit-form-blocks.php#L231.

Reply:
Yes, I followed this logic 👍